lib/xcpretty/formatters/formatter.rb

```diff
@@ -104,11 +105,11 @@ def use_unicode?
     # Will be printed by default. Override with '' if you don't want summary
     def format_test_summary(executed_message, failures_per_suite)
       failures = format_failures(failures_per_suite)
-      if failures.empty?
-        final_message = green(executed_message)
-      else
-        final_message = red(executed_message)
-      end
+      final_message = if failures.empty?
```

Comment:
Maybe use `?:` operator to use less vertical space.

Reply:
rubocop doesn't like multiline ternary